server/web/config.py

```diff
@@ -1,18 +1,40 @@
 import os
 
 class Config():
-    SECRET_KEY = '%yt1uvp5-v52s=+(kt)fan$nne$b4s5u(+*o)b(0cd_)hkzp30';
+    SECRET_KEY = '%yt1uvp5-v52s=+(kt)fan$nne$b4s5u(+*o)b(0cd_)hkzp30'
+
+    DB_USER = "app"
+    DB_HOST = 'db'
+    DB_PASS = 'pass'
+
     def init_app(app):
         pass
 
 class DevelopmentConfig(Config):
+    DB_USER = "app"
+    DB_HOST = 'db'
+    DB_PASS = 'pass'
+    DB_NAME = 'Development'
     DEBUG = True
+    SQLALCHEMY_DATABASE_URI = 'mysql://{user}:{pw}@{host}/{db}'.format(
```

Comment:
Use an [`fstring`](https://www.python.org/dev/peps/pep-0498/) given we are running python 3.6

```python
SQLALCHEMY_DATABASE_URL = f'mysql://{self.DB_USER}:{self.DB_PASS}@{self.DB_HOST}/{self.DB_NAME}'
```